apikey-service/src/main/java/eu/europeana/apikey/Application.java

```diff
@@ -118,11 +92,10 @@ public void configure(WebSecurity webSecurity) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http    .authorizeRequests().anyRequest().access("hasRole('ROLE_ADMIN')")
+        http    .authorizeRequests().anyRequest().authenticated()
                 .and().httpBasic()
                 .and().csrf().disable();
```

Comment:
> isn't csrf().disable() disabling the CORS filtering?

I guess so. But why it was disabled is rather a question to Maike...I left it unchanged

Reply:
yes indeed, that code existed before, I just added the comment becasue CORS support is required through the specs. I didn't tested it, but I would assume that it doesn't work propertly with this configuration. (it might have been issues with the swagger console for which that cors were disabled)